### PR TITLE
Update rubygem-hammer_cli_foreman_remote_execution to 0.1.2

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/plugins/rubygem-hammer_cli_foreman_remote_execution/rubygem-hammer_cli_foreman_remote_execution.spec [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
-Summary: Foreman Remote Execution commands for Hammer CLI
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.1.1
+Version: 0.1.2
 Release: 1%{?foremandist}%{?dist}
-Group: Applications/System
-License: GPLv3
+Summary: CLI for the Foreman remote execution plugin
+Group: Development/Languages
+License: GPL-3.0-or-later
```
Comment (ekohl) [inline packages/plugins/rubygem-hammer_cli_foreman_remote_execution/rubygem-hammer_cli_foreman_remote_execution.spec:16]: This is the SPDX license tag, not the RPM one.

Comment (evgeni) [reply packages/plugins/rubygem-hammer_cli_foreman_remote_execution/rubygem-hammer_cli_foreman_remote_execution.spec]: tell `licenses.json` (like I did now ;P)

File: licenses.json [hunk 2]
```diff
@@ -2,5 +2,6 @@
   "GPL-2.0-or-later": "GPLv2+",
   "GPL-3.0": "GPLv3",
   "GPL-3.0+": "GPLv3+",
-  "GPLv3-or-later": "GPLv3+"
+  "GPLv3-or-later": "GPLv3+",
```
Comment (ekohl) [inline licenses.json:5]: Perhaps this should be a separate commit.

Comment (evgeni) [reply licenses.json:5]: it can be, if you want to. but still noting that my addition is a valid spdx.

Comment (ekohl) [reply licenses.json:5]: Yes. I think it should be a separate commit. I'd be OK with just adding a line but let's keep it sorted.

Comment (evgeni) [reply licenses.json:5]: updated
